packages/remix-gcloud-functions-gen2/server.ts

```diff
@@ -0,0 +1,164 @@
+import type { NextFunction } from "express";
+import type { AppLoadContext, ServerBuild } from "@remix-run/server-runtime";
```

Comment:
```suggestion
import type { AppLoadContext, ServerBuild } from "@remix-run/node";
```

Reply:
`@remix-run/server-runtime` is the implementation detail of `@remix-run/{cloudflare,deno,node}`, which includes common functionality.

Specific runtimes either export the functionality directly _or_ create a custom version of it, so we should always use a specific runtime as a user.